### DISCUSSION TITLE
When to expect my merged pull requests to be available in Assist?

### DISCUSSION THREAD

**Discussion by Ahmed-farag36** [OHF-Voice/intents#1319] [_Q&A] [answered]
Hello guys,
I merged a pull request in my language (ar) and have tested it with command line, when should I be able to test it with Assist panel in Home Assistant dashboard?
In another words, when new voice releases added to home assistant updates?
And would I expect it in core, OS or supervisor updates?

**Comment by TheFes** [comment, marked as answer]
It's added in core updates, and only in the major (.0) update.
The nightly release which is done each night also includes the new intents

**Reply by chatziko** [reply]
Is there a reasonable way (even hackish) to use the latest intents from the github repo in an active HA installation? It would be quite useful for testing.

**Reply by tetele** [reply]
Not in a production-grade installation. If you're willing to clone the HA Core repo and run it yourself in a sandbox, everything's possible.
